### PR TITLE
Form explainer 'new' tag

### PR DESCRIPTION
**Pull request by cfarm**
Add 'new' tag element to the form explainer content boxes.

## Additions

- Logic for new tags
- 'new' property on loan estimate's first content box - as example til we have the actual content that needs these.

## Removals

- alert tab styles - we removed the alert tabs from the design so these are no longer needed.

## Changes

- refactored the conditionals for the term (aka content box) so that there is less duplicate stuff

## Testing

- go to [http://localhost:7000/loan-estimate/](http://localhost:7000/loan-estimate/) to see the New tag output on the first content box.
- CMD+F5 to use voiceover and tab through the page to hear the tag read aloud (it's after the content title, to give the user context).

## Review

- @sonnakim 
- @virginiacc 

### Small screen
screen shot 2015-07-15 at 1 44 48 pm

### Large screen
screen shot 2015-07-15 at 1 44 56 pm

### CODE REVIEW

**Comment by sonnakim** [comment]
@cfarm Is that Avenir regular weight italic, 11 pt?

**Comment by cfarm** [comment]
"AvenirNextLTW01-Italic", Arial, sans-serif (i used the existing ['micro-copy' style](https://cfpb.github.io/cf-typography/docs/) from Capital Framework)

**Comment by sonnakim** [comment]
Cool, just wanted to double-check—the web font as rendered looks slightly thicker than it did in the PSD. 

I think this is good to go :+1:

File: src/loan-estimate/loan-estimate-1.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
         "left": "16.17%",
         "top": "15.02%",
         "width": "21.56%",
-        "height": "4.50%"
+        "height": "4.50%",
+        "new": true
```
Comment (virginiacc) [inline src/loan-estimate/loan-estimate-1.html]: What if we called this property something like 'descriptionTag' and made 'new' its value? That would make it possible for us to apply different tags in future, if we ever wanted to do so...

**Comment by virginiacc** [comment]
@cfarm Looks great! :+1: I had one minor suggestion about making the tagging reusable, but I think this is ready to go when the content is done.

File: src/loan-estimate/loan-estimate-1.html (hunk 1 above)
Comment (cfarm) [inline src/loan-estimate/loan-estimate-1.html]: good idea! I'll add it.

**Comment by cfarm** [comment]
@virginiacc I made the change you suggested.

**Comment by cfarm** [comment]
No longer needed! @!!!!

**Pull request closed without merging**